>>> from exceptional import string_log
>>> string_log("two five".split())
3.2188758248682006
>>> string_log("cat dog".split())
Conversion error: KeyError('cat')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/abingham/repos/sixty-north/pluralsight/core-python-getting-started/resources/m08/visning/clip-05/use-convert-v11/exceptional.py", line 31, in string_log
    v = convert(s)
  File "/Users/abingham/repos/sixty-north/pluralsight/core-python-getting-started/resources/m08/visning/clip-05/use-convert-v11/exceptional.py", line 22, in convert
    number += DIGIT_MAP[token]
KeyError: 'cat'
>>> string_log(8675309)
Conversion error: TypeError("'int' object is not iterable")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/abingham/repos/sixty-north/pluralsight/core-python-getting-started/resources/m08/visning/clip-05/use-convert-v11/exceptional.py", line 31, in string_log
    v = convert(s)
  File "/Users/abingham/repos/sixty-north/pluralsight/core-python-getting-started/resources/m08/visning/clip-05/use-convert-v11/exceptional.py", line 21, in convert
    for token in s:
TypeError: 'int' object is not iterable
>>> 7[12;1H[38;5;0m[48;5;214mConversion error87[12;1H[0mConversion error8
